Research/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -709,8 +709,13 @@ open class RSDStepViewController : UIViewController, RSDStepController, RSDCance
     open func shouldConfirmCancel() -> Bool {
         return self.stepViewModel.rootPathComponent.shouldConfirmCancel()
     }
-    
-    /// Finish canceling the task. This is called once the cancel is confirmed by the user.
+	
+	/// Hook to let tasks know that user decided not to cancel.
```

Comment:
Super minor, but what do you think about specifically calling out in the documentation comments that this does nothing by default and should be overridden by subclasses to called back when the alert action is canceled?